### DISCUSSION TITLE
掃雷更安全，美軍無人獵雷艇通過爆震測試

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#178] [General]
這項稱為無人操作掃雷系統（Unmanned Influence Sweep System，UISS）的獵雷艇，是德士隆系統（Textron System）通用水面無人載具（Common Unmanned Surface Vehicle，CUSV）的其中一款設定，除了 UISS 之外，CUSV 還可以依照其他任務型式，安裝機砲、水雷、C-RAM 或其他武器和偵測裝備。
https://technews.tw/2022/01/05/us-navys-new-minesweeping-uiss-passed-shock-trials/

**Comment by cobb110202008** [comment]
無人機或無人載具在軍事上的運用已經逐漸成熟，若是用在戰爭上，戰爭恐怕會更慘烈。

**Comment by briankuo123** [comment]
會不會以後戰爭就變成在一個特定的區域用各自的無人機去打仗，然後決定勝敗阿

**Comment by minniewei** [comment]
真心希望不要用在真的戰爭上(要不還會有人活著嗎....)

**Comment by wh20502** [comment]
感覺之後發明新的以後退役就會賣台灣來賺錢了，除非哪邊戰爭打起來...但我真的不希望

**Comment by Shiba010** [comment]
希望ai不要來作為各國打仗的工具，受苦的往往是平民老百姓...

**Comment by ghost** [comment]
AI不應該拿來兩國廝相殘殺的工具 而是拿來造福人類的好東西才對~

**Comment by Lynn12378** [comment]
科技進步常常會帶來原本預期以外的結果，真的是取決於使用者的心態

**Comment by yjpxi** [comment]
但是我覺得比起傳統掃雷作業，這種無人系統的引入大大提升了海軍在執行掃雷和其他高風險任務中的效率和安全性。由於不需要人員直接操作，UISS能夠在危險的水域中進行掃雷作業，減少了人員傷亡的風險。
